sql/derived_tables/po_lines_tags.sql

```diff
@@ -0,0 +1,17 @@
+DROP TABLE IF EXISTS folio_reporting.po_lines_tags;
+
+-- Create a local table for tags in po_lines.
+
+CREATE TABLE folio_reporting.po_lines_tags AS
+SELECT
+    pol.id AS pol_id,
+    tags #>> '{}' AS pol_tag
+FROM
+    po_lines AS pol
+    CROSS JOIN json_array_elements(
+        json_extract_path(data, 'tags', 'tagList')) AS tags;
```

Comment:
This all looks good to me, but could I request adding the "ordinality" as in https://github.com/library-data-platform/ldp/blob/main/doc/User_Guide.md#7-json-arrays in case it could be useful?

Reply:
I don't know if you are able to set a in a chronology, but perhaps @sbeltaine or Scott might say something on that? 
If it's needed I'm happy to add it.

Reply:
Chronology is not needed for tags.

Reply:
Looks great!

Reply:
Array order is now preserved:
https://github.com/folio-org/folio-analytics/pull/285/commits/8f40bf3e7da41681166257dfe3435548137ef083